### PR TITLE
DEV: Lint files to satisfy core standards.

### PR DESCRIPTION
**Pull request by romanrizzi**
Before moving this plugin into core, we must comply with the their linting rules and Ember code standards.

This change applies two Ember codemods to update hbs templates: `ember-no-implicit-this-codemod` and `ember-angle-brackets-codemod`.

### CODE REVIEW

File: .template-lintrc.js [hunk 1]
```diff
@@ -1,4 +1,8 @@
 module.exports = {
   plugins: ["ember-template-lint-plugin-discourse"],
   extends: "discourse:recommended",
+
+  rules: {
+    "no-capital-arguments": false, // TODO: we extensively use `args` argument name
```
Comment (jjaffeux) [inline .template-lintrc.js:6]: what case do we have with capital arg?

**Comment by jjaffeux** [comment]
I tried it locally and it seems to work just fine, let's just merge this.

File: .template-lintrc.js (hunk 1 above)
Comment (romanrizzi) [reply .template-lintrc.js:6]: The plugin outlet component uses the `@args` attribute, and this rule complains about it. It's already disabled in core, so I'm doing the same here.

Comment (jjaffeux) [reply .template-lintrc.js:6]: hmmm, it's not supposed to be the case AFAIK, this rule is preventing this: `<Foo @Bar={1}>`

https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-capital-arguments.md

Comment (romanrizzi) [reply .template-lintrc.js:6]: It also complains about the `@args` attribute being restricted. See [here](https://github.com/discourse/discourse-chat/actions/runs/3323743368/jobs/5494479287#step:8:11)

**Pull request merged**